homeassistant/components/local_ip/manifest.json

```diff
@@ -3,5 +3,6 @@
   "name": "Local IP Address",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/local_ip",
-  "codeowners": ["@issacg"]
+  "codeowners": ["@issacg"],
+  "quality_scale": "internal"
```

Comment:
This is not internal, it even has an external code owner

Reply:
```suggestion
```